src/wireviz/Harness.py

```diff
@@ -344,6 +345,44 @@ def create_graph(self) -> Graph:
             dot.node(cable.name, label=f'<\n{html}\n>', shape='box',
                      style=style, fillcolor=translate_color(bgcolor, "HEX"))
 
+        def typecheck(name: str, var, type) -> None:
+            if not isinstance(var, type):
+                raise Exception(f'Unexpected value type of {name}: {var}')
+
+        # TODO?: Differ between override attributes and HTML?
+        if self.tweak.override is not None:
+            typecheck('tweak.override', self.tweak.override, dict)
+            for k, d in self.tweak.override.items():
+                typecheck(f'tweak.override.{k} key', k, str)
+                typecheck(f'tweak.override.{k} value', d, dict)
+                for a, v in d.items():
+                    typecheck(f'tweak.override.{k}.{a} key', a, str)
+                    typecheck(f'tweak.override.{k}.{a} value', v, str)
+
+            # Override generated attributes of selected entries matching tweak.override.
+            for i, entry in enumerate(dot.body):
+                if isinstance(entry, str):
+                    # Find a possibly quoted keyword after leading TAB(s) and followed by [ ].
+                    match = re.match(r'^\t*(")?((?(1)[^"]|[^ "])+)(?(1)") \[.*\]$', entry, re.S)
+                    keyword = match and match[2]
+                    if keyword in self.tweak.override.keys():
+                        for attr, value in self.tweak.override[keyword].items():
+                            if len(value) == 0 or ' ' in value:
+                                value = value.replace('"', r'\"')
+                                value = f'"{value}"'
+                            # TODO?: If value is None: delete attr, and if attr not found: append it?
+                            entry = re.sub(f'{attr}=("[^"]*"|[^] ]*)', f'{attr}={value}', entry)
+                        dot.body[i] = entry
+
+        if self.tweak.append is not None:
+            if isinstance(self.tweak.append, list):
```

Comment:
Is there any reason that the `append` section should be a list, and not a string?
Is there a benefit to having multiple list entries for different tweaks, as opposed to all of them concatenated into one string? I am not opposed to it, just genuinely curious.

Reply:
Initially, I used a list of quoted strings to easily specify leading spaces where I wanted them. Leading spaces in multiline strings seems to be stripped by the YAML parser, but maybe you know a way to avoid that?

Reply:
If you include the indentation indicator ([source](https://yaml-multiline.info/)), any additional leading spaces are preserved. I quickly tested this and it works. Example:

```yaml
tweak:
  append: |2  # two leading spaces will be interpreted as indentation; any more will be preserved
              // Tweaking two cable nodes to keep close to each other
              subgraph cluster1 {
                color=yellow // Same as background to hide the surrounding frame
                TPPinkSilver
                TPBlueSilver
              }
```